client/src/components/Navigation/Navigation.tsx

```diff
@@ -48,20 +52,27 @@ const Navigation: React.FC<Props> = ({ user, setUser }) => {
         {user ? (
           <>
             <SidebarWrapper>
-              <NavIcon to="/income">
+              <NavIcon to="/budget">
+                <h2>My Budget</h2>
+              </NavIcon>
+            </SidebarWrapper>
+
+            {/* 
+            <SidebarWrapper>
+              <NavIcon to="/budget/income">
                 <h2>Income</h2>
```

Comment:
Is there a reason for all the commented out code?

Reply:
Yeah I just left it there just in case we wanted to leave it the way it was. If everyone's good with the changes I made- then Ill just go ahead and delete all that code